lessons/lesson2.md

```diff
@@ -1,129 +1,80 @@
-## Lesson 2: Write your app
+## Lesson 2: Set up Project
```

Comment:
not sure I understand the flow in this lesson 2:
* am I supposed to run these in a Firefly project?
* am I supposed to run them in any node js project?